README.md

```diff
@@ -23,10 +23,12 @@ This will start both L0 and L1 clusters on kubernetes using current kube-context
 Initial validators for L0 and L1 have their public ports mapped to local ports 9000 and 9001 respectively.
 
 ```
+curl localhost:9000/cluster/info
 curl localhost:9100/cluster/info
```

Comment:
9000 + 9001 or 9000 + 9100 ?

Reply:
Can you explain what you mean by this question?

Reply:
9000 is mapped to port 9000 of initial L0 validator
9100 is mapped to port 9000 of initial L1 validator

Reply:
Ok, above there is:
> Initial validators for L0 and L1 have their public ports mapped to local ports 9000 and 9001 respectively.

I misunderstood :+1: all fine

Reply:
Wait, you're right. It's an issue in the documentation. It should say:

```
L0 and L1 have their public ports mapped to local ports 9000 and 9100 
```